src/platforms/ios/ioslogger.swift

```diff
@@ -35,46 +37,76 @@ public class IOSLoggerImpl : NSObject {
         )
     }
 
-    @objc func debug(message: String) {
+    @objc func debugToConsole(message: String) {
+        os_log("%{public}@", log: self.log, type: .debug, message)
+    }
+
+    @objc func infoToConsole(message: String) {
+        os_log("%{public}@", log: self.log, type: .info, message)
+    }
+
+    @objc func errorToConsole(message: String) {
+        os_log("%{public}@", log: self.log, type: .error, message)
```

Comment:
nit: To avoid code repetition, maybe we can extract the `os_log` call into a separate function or keep using the `log` fucntion and rip out the part that write to the log file into a separate function? Maybe we can just keep using `debug`, `info` and `error` but pass `writeToLogFile: bool` as an argument.

Reply:
I removed some of the repetition by creating a more general function to go from obj-c to swift. I was having trouble finding a way to use the iOS enum in obj-c code, so I made my own after spending time researching/attempting. (Let me know if you have a code snippet that works for it.)